source/html/js/app/mappers/connections/medialive_input_channel.js

```diff
@@ -4,12 +4,33 @@
 import * as server from "../../server.js";
 import * as connections from "../../connections.js";
 
+function checkAdditionalConnections(results, connection) {
```

Comment:
Should we make a checkAdditionalConnections util file? There is some reused code between these functions

Reply:
the `checkAdditionalConnections` function is refactored out into its own helper function. Also updated the files where the function is used